src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const StateCopy = { ...state };
+  const EmptyArray = [];
```

Comment:
Variable names should follow camelCase naming convention. Consider renaming 'EmptyArray' to 'emptyArray'.